pages/renew.tsx

```diff
@@ -167,6 +157,80 @@ export default function Renew() {
     }
   }, [activeStep, isReviewing]);
 
+  const handleCompletePersonalInformationStep = (values: {
+    updatedAddress: boolean;
+    personalAddressLine1: string;
+    personalAddressLine2: string;
+    personalCity: string;
+    personalPostalCode: string;
+  }) => {
+    setUpdatedAddress(!!Number(values.updatedAddress));
+    if (values.updatedAddress) {
+      setPersonalAddressLine1(values.personalAddressLine1);
+      setPersonalAddressLine2(values.personalAddressLine2);
+      setPersonalCity(values.personalCity);
+      setPersonalPostalCode(values.personalPostalCode);
+    }
```

Comment:
Should set these to `null` if `updatedAddress` is false? Applies to the other sections as well